cate/core/opimpl.py

```diff
@@ -650,12 +654,31 @@ def _get_geo_spatial_cf_attrs_from_var(ds: xr.Dataset, var_name: str, allow_poin
                 dim_res = abs(var.values[1] - var.values[0])
                 dim_min = min(var.values[0], var.values[-1]) - 0.5 * dim_res
                 dim_max = max(var.values[0], var.values[-1]) + 0.5 * dim_res
-            elif len(var.values) == 1 and allow_point:
-                dim_var = var
-                # Actually a point with no extent
-                dim_res = 0.0
-                dim_min = var.values[0]
-                dim_max = var.values[0]
+            elif len(var.values) == 1:
+                if res_name in ds.attrs:
+                    dim_res = ds.attrs[res_name]
+                    if isinstance(dim_res, str):
+                        # remove any units from string
+                        dim_res = dim_res.replace('degree', '').replace('deg', '').replace('°', '').strip()
```

Comment:
What about `'degrees'` or, also commonly used, `'decimal degrees'` and so forth?

As I said, I'd not take this information from metadata attributes. If we have coordinate data, take it from there! There is code in cate or xcube that you can (re)use for this purpose.

Reply:
The reason why we are taking this information from the metadata is because we can not derive it from the coordinates. This is our fallback solution to determine the boundaries in case there is only one entry available for the dimension.

I think it makes sense to state explicitly which names we want to filter out, as sometimes the unit might be 'km' or something else. In that case, I'd rather not use this piece of information.